tensorflow_lite_support/python/task/processor/proto/class_pb2.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Class options protobuf."""
+
+from tensorflow_lite_support.cc.task.vision.proto import class_pb2
+
+Class = class_pb2.Class
```

Comment:
@lu-wang-g @mbrenon It looks like `Class` is called `Category` in both the C (iOS) and Java API. Should we do the same for the Python API?
https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/c/task/processor/category.cc
https://www.tensorflow.org/lite/api_docs/java/org/tensorflow/lite/support/label/Category

Reply:
Yes, let's maybe go for consistency across wrappers where possible and call that `Category` here too. The `Class` naming is historical and we can hardly get rid of it in the C++ API, but with hindsight tends to cause a lot of problems because it's usually a reserved keyword...